### PR TITLE
Resolves #157 About Page Changes

### LINKED ISSUE #157: Update About Page
Change text on the page and add an image below the App/Dev gold image.

Eventually link the image to a blog post Chuck makes on the MCSE certification.

### PR DESCRIPTION
**Pull request by kenhowardpdx**
@JamieMikami Here's the PR. Branch is `157-about-page`

### CODE REVIEW

**Comment by JamieMikami** [comment]
That looks better, thanks

**Comment by JamieMikami** [comment]
:shipit: if it looks good to you.

File: server/views/page-about.html [hunk 1]
```diff
@@ -12,12 +12,27 @@
         <div class="sticky" data-sticky data-anchor="content-container">
         {{> contact-widget}}
         {{> posts-widget}}
+        <!--img src="/resources/images/site/mvp-logo.png" alt="Microsoft Most Valuable Professional" /><p> </p-->
```
Comment (kenhowardpdx) [inline server/views/page-about.html]: For a future update?

File: server/views/page-about.html [hunk 2]
```diff
@@ -12,12 +12,27 @@
...
+        <img src="/resources/images/site/mcse-charter.png" alt="Microsoft Certified Solutions Expert" /><p> </p>
         <img src="/resources/images/site/ms-gold-logo.png" alt="Microsoft Gold Certified Partner" />
         </div>
     </div>
     <div class="medium-8 large-9 columns" id="content-container">
         <article>
-            {{{markdown pageContent}}}
+<p class="paragraph-primary">CSG Pro builds modern software solutions that expand knowledge through a more
```
Comment (kenhowardpdx) [inline server/views/page-about.html]: Nitpick - Indentation.

**Pull request merged**